### DISCUSSION TITLE
[nuxt-bridge] CI tests dependent on .nuxt directory

### DISCUSSION THREAD

**Discussion by Davounet** [nuxt/nuxt#17753] [Questions] [answered]
Hello !

I am migrating a typescript nuxt2 project to nuxt-bridge, and can not get our CI tests to run.
The typescript configuration is dependent on the one generated in the `.nuxt` directory, which is not built in CI, so everything results in this error : 
```
error TS5083: Cannot read file '/home/runner/work/app/app/front/.nuxt/tsconfig.json'.
```
I am trying to generate the `.nuxt` directory ahead of the tests but would like to avoid generating/building the whole project for this.
Another way to solve this would be to migrate from jest to vitest but this looks like a lot of work so I am looking for other solutions first.

Has anyone a idea on how to handle this situation ?
Thank you, have a nice day !

**Comment by ninebolt6** [comment, marked as answer]
You can generate `.nuxt` with `npx nuxi prepare` command.

**Reply by Davounet** [reply]
Thank you !
